revista-lecciones-y-ensayos.csl

```diff
@@ -0,0 +1,420 @@
+  <!-- This style was edited with the Visual CSL Editor (https://editor.citationstyles.org/visualEditor/) -->
+  <info>
+    <title>Revista Lecciones y Ensayos (español)</title>
+    <title-short>revistalye</title-short>
+    <id>http://www.zotero.org/styles/revista-lecciones-y-ensayos</id>
+    <link href="http://www.zotero.org/styles/revista-lecciones-y-ensayos" rel="self"/>
+    <link href="http://www.zotero.org/styles/the-journal-of-comparative-neurology" rel="template"/>
+    <link href="http://revistas.derecho.uba.ar/index.php/revistalye/about/submissions" rel="documentation"/>
+    <author>
+      <name>Tomás D'Alessio</name>
+      <email>tomasdalessiod@gmail.com</email>
+    </author>
+    <author>
+      <name>Andrés D'Alessio</name>
+    </author>
+    <category citation-format="note"/>
+    <category field="law"/>
+    <issn>0024-0079</issn>
+    <eissn>2362-4620</eissn>
+    <summary>Style for Revista Lecciones y Ensayos</summary>
+    <updated>2025-04-21T19:53:01+00:00</updated>
+    <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
+  </info>
+  <locale xml:lang="es">
+    <terms>
+      <term name="et-al">y otros</term>
+      <term name="opus" form="short">considerando</term>
+    </terms>
+  </locale>
+  <macro name="editor">
+    <names variable="editor" font-variant="normal">
+      <name delimiter=" " and="symbol" delimiter-precedes-last="always" initialize="false" initialize-with="" name-as-sort-order="all" sort-separator=" ">
+        <name-part name="family" font-variant="small-caps" suffix=", "/>
+      </name>
+      <label form="short" text-case="lowercase" prefix=" (" suffix=")"/>
+    </names>
+  </macro>
+  <macro name="jurisdiccion-ente">
+    <names variable="author" font-variant="normal"/>
+  </macro>
+  <macro name="contributor">
+    <names variable="contributor" font-variant="normal"/>
+  </macro>
+  <macro name="author">
+    <group>
+      <names variable="author" font-variant="normal">
+        <name font-variant="normal" and="symbol" delimiter-precedes-et-al="never" delimiter-precedes-last="never" initialize="false" initialize-with="." name-as-sort-order="all">
+          <name-part name="family" font-variant="small-caps"/>
+        </name>
+        <label form="short" text-case="lowercase" strip-periods="true" font-variant="normal" prefix=" "/>
+        <substitute>
+          <names variable="editor"/>
+        </substitute>
+      </names>
+    </group>
+  </macro>
+  <macro name="author-short">
+    <names variable="author">
+      <name form="short" and="symbol" delimiter-precedes-et-al="never" delimiter-precedes-last="never" et-al-use-first="3" initialize-with=".">
+        <name-part name="family" font-variant="small-caps"/>
+      </name>
+      <substitute>
+        <names variable="editor"/>
+        <names variable="translator"/>
+      </substitute>
```

Comment:
Add a `<text variable="title"/>` or similar to the "substitute" to handle items without an author/editor/translator